### PR TITLE
adding a second pretty print algebra 

### PR DESCRIPTION
**Pull request by sjanssen2**
to better illustrate binding and missing partners in buldges. Inspired by Sophie Stebel.

basically, the same candidate is no longer shown like
```
A     UG A       AC        
 GGAGU  C  CCAGGU  AGCUGU  
 CUUCA  G  GGUCCG  UCGACG  
      GU AG              UU
```
but now as
```
target 5' A        A-      AC         3'
           GGAGUUGC  CCAGGU  AGCUGU     
           |:|||::|  |||||:  |||||:     
           CUUCAGUG  GGUCCG--UCGACG     
miRNA 3'           AG              UU 5'

```

### CODE REVIEW

File: rnahybrid.gap
Comment (Anna-Rehm) [inline rnahybrid.gap:1]: Do you have any place where you explain how the programming/syntax in rnahybrid works? Maybe some comments that go into more detail than the comment at the bottom would be beneficial :)

**Review by Anna-Rehm — APPROVED** [review]

File: rnahybrid.gap
Comment (sjanssen2) [reply rnahybrid.gap:1]: the syntax is what the poor students have to learn in ADP. Not sure if this is the right place to elaborate on it.

Comment (Anna-Rehm) [reply rnahybrid.gap:1]: Got it! Yeah I agree, that's not the right place then.

**Review by kmaibach — APPROVED** [review]
This is a great idea and makes it so much easier to see the structure at first glance.

**Pull request merged**